wire-ios-sync-engine/Source/UserSession/ZMUserSession/Dependencies/UserSessionDependencies.swift

```diff
@@ -18,8 +18,13 @@
 
 final class UserSessionDependencies {
     var caches: Caches
+    var analyticsSessionConfiguration: AnalyticsSessionConfiguration?
 
-    init(caches: Caches) {
+    init(
+        caches: Caches,
+        analyticsSessionConfiguration: AnalyticsSessionConfiguration? = nil
```

Comment:
I suggest to not provide a default value.
```suggestion
        analyticsSessionConfiguration: AnalyticsSessionConfiguration?
```